src/@next/components/molecules/Thumbnail/Thumbnail.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 
 import { maybe } from "@utils/misc";
 
+import { PlaceholderImage } from "@components/atoms/PlaceholderImage";
```

Comment:
Please add `PlaceholderImage` to the list inside the `src/@next/components/atoms/index.ts` file.
```suggestion
import { PlaceholderImage } from "@components/atoms";
```